extension/package.json

```diff
@@ -10,7 +10,7 @@
     "url": "https://blswallet.org"
   },
   "engines": {
-    "node": ">=10.0.0",
+    "node": ">=12.0.0",
```

Comment:
Might be safe to go to v14 or v16 (LTS) as well https://nodejs.org/en/about/releases/

Reply:
This was fun... so lts is actually v18 now so I tried that.

This broke webpack because... openssl removed support for md4 and therefore node removed it, but webpack was using it: https://github.com/webpack/webpack/blob/v4.46.0/lib/NamedModulesPlugin.js#L11 😱😱😭.

I don't suppose webpack needs a secure hash function, but, considering md4 was broken in 1995 - long before webpack or nodejs for that matter, it's amazing that webpack would choose md4. Even md5 would be less surprising.

Anyway, I guess we're stuck on node 16 until we upgrade to webpack 5 😂.

Reply:
done